lxd/network/acl/acl_ovn.go

```diff
@@ -1153,3 +1155,56 @@ func ovnParseLogEntry(input string, prefix string) string {
 
 	return string(out)
 }
+
+// ovnParseLogEntriesFromJournald reads the OVN log entries from the systemd journal and returns them as a list of string entries.
+func ovnParseLogEntriesFromJournald(systemdUnitName string, prefix string) ([]string, error) {
```

Comment:
this function isnt used in the changes of this commit, so feels like this should be separated into 2 commits, or have this function be added as part of a later commit.

This commit should just be concerned with changing the function signature of `ovnParseLogEntry`.